packages/components/src/modal/style.scss

```diff
@@ -134,17 +133,46 @@
 }
 
 // Scale down the content of the page.
-body.modal-open.is-fullscreen-modal > *:not(.components-modal__portal) {
-	transition: transform 0.5s;
-	transform: scale(0.9);
+body.modal-open.is-fullscreen-modal {
+	background: $gray-900;
+
+	> *:not(.components-modal__portal) {
+		animation: components-modal__content-scale-back 0.4s cubic-bezier(0.165, 0.84, 0.44, 1) forwards;
+	}
+
+	.interface-interface-skeleton__editor {
```

Comment:
This is not something we can target here, we don't know what's rendered on the page on a generic component like "Modal"

Reply:
I'll remove 👍